README.md

```diff
@@ -146,7 +147,7 @@ The table below describes all the available configuration values for Waggle Danc
 | `primary-meta-store.name`                             | Yes      | Database name that uniquely identifies this metastore used internally. Cannot be empty. |
 | `primary-meta-store.database-prefix`                  | No       | This will be ignored for the primary metastore and an empty string will always be used instead. |
 | `primary-meta-store.access-control-type`              | No       | Sets how the client access controls should be handled. Default is `READ_ONLY` Other options `READ_AND_WRITE_AND_CREATE`, `READ_AND_WRITE_ON_DATABASE_WHITELIST` and `READ_AND_WRITE_AND_CREATE_ON_DATABASE_WHITELIST` see Access Control section below. |
-| `primary-meta-store.writable-database-white-list`     | No       | White-list of databases used to verify write access used in conjunction with `primary-meta-store.access-control-type`. The list of databases should be listed without any `primary-meta-store.database-prefix`. |
+| `primary-meta-store.writable-database-white-list`     | No       | White-list of databases used to verify write access used in conjunction with `primary-meta-store.access-control-type`. The list of databases should be listed without any `primary-meta-store.database-prefix`. This property supports both full database names and RegEx patterns. Further information on Java RegEx patterns can be found [here](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html)|
```

Comment:
Full stop

Reply:
and [Java RegEx patterns](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html).|